packages/apollo-server-adonis/src/adonisApollo.ts

```diff
@@ -35,7 +35,8 @@ export function graphqlAdonis(
       query,
     }).then(
       gqlResponse => {
-        response.json(gqlResponse);
+        response.header('Content-Type', 'application/json');
```

Comment:
Not familiar with Adonis patterns but, aside from being more concise, is there a benefit to using the `response.type('application/json')` technique (outlined in the [docs here](https://adonisjs.com/docs/4.0/response#_headers)) as opposed to this more explicit `response.header`?

(Keep in mind that we've never supported Adonis 3.0, so the fact that it isn't present in v3 docs shouldn't matter.)